### PR TITLE
[FIX] error: invalid command 'bdist_wheel' #226

### PR DESCRIPTION
**Pull request by aidoskz**
[FIX] error: invalid command 'bdist_wheel' #226

### CODE REVIEW

File: 11.0/Dockerfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ RUN set -x; \
         && rm -rf /var/lib/apt/lists/* odoo.deb
 
 # Copy entrypoint script and Odoo configuration file
+RUN pip3 install wheel
```
Comment (ecrazor1911) [inline 11.0/Dockerfile]:
```Dockerfile
RUN pip3 --no-cache-dir install wheel \
 && pip3 --no-cache-dir install num2words xlwt
```

Comment (aidoskz) [inline 11.0/Dockerfile]: @ecrazor1911 Thanks! Resolved! 👍

Comment (aidoskz) [reply 11.0/Dockerfile]: @ecrazor1911 Thanks! Resolved! 👍

**Comment by aidoskz** [comment]
@d-fence i've resolved conflict

**Comment by d-fence** [comment]
* num2words and xlwt are already installed
* I don't see the problem that adding "bdist_wheel" tries to solves ?

**Comment by aidoskz** [comment]
ok i've close this PR

**Pull request closed without merging**